### PR TITLE
feat: add `verify-ens` cmd utility

### PR DESCRIPTION
**Pull request by osmaczko**
Build&Run: `make verify-ens && ./build/bin/verify-ens`

A very simple example of how to use ens verifier.

### CODE REVIEW

**Comment by osmaczko** [comment]
CC: @felicio

File: cmd/verify-ens/main.go
Comment (felicio) [inline cmd/verify-ens/main.go:1]: The original test file that I've shared with you could be ran and stepped through while debugging just fine. And was enough for my use case.

I'm not sure if turning it into this utility is useful for anyone else. Maybe so if instead it just returned a pk for name via CLI param. But I'm not convinced about that either, so leaving that up to you.

Thanks anyway.

File: cmd/verify-ens/main.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import (
+	"fmt"
+
+	"github.com/status-im/status-go/eth-node/bridge/geth/ens"
+	enstypes "github.com/status-im/status-go/eth-node/types/ens"
+	"go.uber.org/zap"
+)
+
+const infuraToken = "putYourTokenHere"
+const verifyENSContractAddress = "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e"
```
Comment (felicio) [inline cmd/verify-ens/main.go:12]: Not really used.

Comment (osmaczko) [reply cmd/verify-ens/main.go:12]: WDYM? It is used in line 27

File: cmd/verify-ens/main.go
Comment (osmaczko) [reply cmd/verify-ens/main.go:1]: Sorry, I didn't notice the test file you shared with me earlier. It seems that you had edited the previous message, which is why I missed it. I agree that the utility is not useful, the changes I made were solely for illustrative purposes and were not meant to be merged.

File: cmd/verify-ens/main.go (hunk 1 above)
Comment (felicio) [reply cmd/verify-ens/main.go:12]: I meant further down the stack. I didn't know that when looking up the value and later sharing it with you.

**Pull request closed without merging**